test-comments.yml

```diff
@@ -5,20 +5,20 @@
 # Test comments
 test:
   number: 5
+  boolean: true
```

Comment:
The order of setting items should be preserved.

`test.boolean` is set after `test.string` in `YamlTest.java`.